src/library_wasmfs.js

```diff
@@ -203,6 +203,27 @@ FS.createPreloadedFile = FS_createPreloadedFile;
 
 #endif
   },
+
+  // A minimal version of PATH_FS for lookups. This overrides the version in
+  // library_path.js.
+  $PATH_FS: {
```

Comment:
I would expect a corresponding change to library_path.js.. do we still need to include that file?   If not perhaps we can put `#if !WASMFS` around the old code?

Reply:
We do still use some parts of `PATH`, which is generic path operations without FS integration. But good idea to ifdef, I added that now.